tests/StorageBackendTest.php

```diff
@@ -247,6 +247,11 @@ public function testStorageBackendList()
         $this->assertArrayHasKey('host', $backend);
         $this->assertArrayHasKey('username', $backend);
         $this->assertArrayHasKey('backend', $backend);
+
+        $backedDetail = $this->client->getStorageBackend($backend['id']);
+        $this->assertArrayHasKey('host', $backedDetail);
+        $this->assertArrayHasKey('username', $backend);
+        $this->assertArrayHasKey('backend', $backedDetail);
```

Comment:
moc rozumně to nejde otestovat, nestupovat ten saml je taky na prd pro manage aby to uspokojilo test. Prakticky stačí že je to otestované unit testem v connection